src/main/java/org/pac4j/demo/dw/Pac4JDemoConfiguration.java

```diff
@@ -1,14 +1,14 @@
 package org.pac4j.demo.dw;
 
-import org.pac4j.dropwizard.Pac4jFactory;
-
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import io.dropwizard.Configuration;
+import org.pac4j.demo.dw.factory.Pac4jFactory2;
 
 public class Pac4JDemoConfiguration extends Configuration {
 
     @JsonProperty("pac4j")
-    Pac4jFactory pac4jFactory = new Pac4jFactory();
+    //Pac4jFactory1 pac4jFactory = new Pac4jFactory1();
+    Pac4jFactory2 pac4jFactory = new Pac4jFactory2();
```

Comment:
`Pac4jFactory1` and `Pac4jFactory2` are the two main approaches for enhanced configurations: either you provide a configuration class and whatever properties you want (and it's properly handled by the `Pac4jFactory1` and the configuration class) OR you provide specific properties and it's properly handled by the `Pac4jFactory2` using the `PropertiesConfigFactory`.